src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -1030,6 +1017,29 @@ private static void VerifyModelForOutVarInNotExecutableCode(SemanticModel model,
             Assert.Equal(expectedType, typeInfo.Type);
             Assert.Equal(expectedType, typeInfo.ConvertedType);
             Assert.Equal(typeInfo, ((CSharpSemanticModel)model).GetTypeInfo(decl));
+            Assert.True(model.GetConversion(decl).IsIdentity);
```

Comment:
This assertion may be correct for the current set of tests, but I don't think it is correct in general, for example the declaration `int z` in `(long x, long y) = (int z, int w);`. Please add that as a test.